app/renderer/components/Inspector/SelectedElement.js

```diff
@@ -116,65 +121,28 @@ const SelectedElement = (props) => {
   }
 
   // Get the data for the strategies table
-  let findDataSource = _.toPairs(getLocators(attributes, sourceXML)).map(([key, selector]) => ({
+  let findDataSource = selectedElement.strategyMap.map(([key, selector]) => ({
     key,
     selector,
     find: key,
   }));
 
-  // If XPath is the only provided data source, warn the user about it's brittleness
-  let showXpathWarning = false;
-  if (findDataSource.length === 0) {
-    showXpathWarning = true;
-  }
-
-  // Add class chain to the data source as well
-  if (classChain && currentContext === NATIVE_APP) {
-    const classChainText = (
-      <span>
-        -ios class chain
-        <strong>
-          <a onClick={(e) => e.preventDefault() || shell.openExternal(CLASS_CHAIN_DOCS_URL)}>
-            &nbsp;(docs)
-          </a>
-        </strong>
-      </span>
-    );
-
-    findDataSource.push({
-      key: '-ios class chain',
-      find: classChainText,
-      selector: classChain,
-    });
-  }
-
-  // Add predicate string to the data source as well
-  if (predicateString && currentContext === NATIVE_APP) {
-    const predicateStringText = (
-      <span>
-        -ios predicate string
-        <strong>
-          <a onClick={(e) => e.preventDefault() || shell.openExternal(PREDICATE_DOCS_URL)}>
-            &nbsp;(docs)
-          </a>
-        </strong>
-      </span>
-    );
-
-    findDataSource.push({
-      key: '-ios predicate string',
-      find: predicateStringText,
-      selector: predicateString,
-    });
+  // Add documentation links to supported strategies
+  for (const locator of findDataSource) {
+    switch (locator.key) {
+      case '-ios class chain':
+        locator.find = locatorStrategyDocsLink(locator.key, CLASS_CHAIN_DOCS_URL);
+        break;
+      case '-ios predicate string':
+        locator.find = locatorStrategyDocsLink(locator.key, PREDICATE_DOCS_URL);
+        break;
+    }
   }
 
-  // Add XPath to the data source as well
-  if (xpath) {
-    findDataSource.push({
-      key: 'xpath',
-      find: 'xpath',
-      selector: xpath,
-    });
+  // If XPath is the only optimal selector, warn the user about its brittleness
+  let showXpathWarning = false;
```

Comment:
Sometimes this warning might be annoying. How about having a possibility of hiding it forever or at least until the current inspector session finishes?

Reply:
Hmm, I personally don't find it too annoying since it only really appears if only the xpath locator applies. In fact, an unintentional change in this PR is also counting class chain and predicate string towards this length (previously the length was checked _before_ adding these two strategies).